src/initialize-transformed.js

```diff
@@ -279,7 +279,7 @@ async function initializeTransformed ({ $element, layout, component }) {
       },
       options: {
         backgroundColor: colors[`vColLib${layout.ColorSchema}`],
-        backgroundColorOdd: colors[`vColLib${layout.ColorSchemaP}`],
```

Comment:
@kristofferlind probably a novice question but what does the `P` do in this situation?

Reply:
it's just to map to two variables of two shades of colors (i.e  vColLibBlue& vColLibBlueP)